lib/cog_api/client.ex

```diff
@@ -6,15 +6,16 @@ defmodule CogApi.Client do
 
   @callback authenticate(%Endpoint{}) :: {atom, %Endpoint{}}
 
-  @callback group_index(%Endpoint{}) :: {atom, [%Group{}]}
-  @callback group_create(%Endpoint{}, %{}) :: {atom, %Group{}}
-
-  @callback permission_index(%Endpoint{}) :: {atom, [%Permission{}]}
-  @callback permission_create(%Endpoint{}, String.t) :: {atom, %Permission{}}
-
   @callback role_index(%Endpoint{}) :: {atom, [%Role{}]}
   @callback role_show(%Endpoint{}, String.t) :: {atom, %Role{}}
   @callback role_create(%Endpoint{}, %{}) :: {atom, %Role{}}
   @callback role_update(%Endpoint{}, String.t, %{}) :: {atom, %Role{}}
   @callback role_delete(%Endpoint{}, String.t) :: atom
+
+  @callback permission_index(%Endpoint{}) :: {atom, [%Permission{}]}
+  @callback permission_create(%Endpoint{}, String.t) :: {atom, %Permission{}}
+
+  @callback group_index(%Endpoint{}) :: {atom, [%Group{}]}
+  @callback group_show(%Endpoint{}, String.t) :: {atom, %Group{}}
+  @callback group_create(%Endpoint{}, %{}) :: {atom, %Group{}}
```

Comment:
Why were these moved below? It seems like they were sorted alphabetically before